frontend/src/index.tsx

```diff
@@ -4,6 +4,10 @@ import './index.css';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
 import './fonts/SawarabiGothic-Regular.ttf';
+import { BackendMode } from './common/Utils';
+import 'bootstrap/dist/css/bootstrap.min.css';
```

Comment:
Do not change this file. We have two indexes:
`indexWeb3.tsx`
and
`indexUnifyre.tsx`

Then there is a `setup.sh` that will copy one of them over `index.ts`.
This is to build the project for desktop or unifyre separately.